### PR TITLE
SWIP-3 Support RocketMQ monitoring

### PR DESCRIPTION
**Pull request by peachisai**
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -50,5 +50,6 @@
 * Update the `Overview` docs to add the `Service Hierarchy Relationship` section.
 * Fix incorrect words for `backend-bookkeeper-monitoring.md` and `backend-pulsar-monitoring.md`
 * Document a new way to load balance OAP.
+* Add `SWIP-3 Support rocketmq monitoring`.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:53]:
```suggestion
* Add `SWIP-3 Support RocketMQ monitoring`.
```

File: docs/en/swip/SWIP-3.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### RocketMQ Broker Supported Metrics
+
+| Monitoring Panel                           |Unit        | Metric Name                                                             | Description                                                                                            | Data Source      |
+|--------------------------------------------|------------|-------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|------------------|
+| Produce Tps                                | Msg/sec    | meter_rocketmq_broker_produce_tps                                       | The number of broker produces messages per second.                                                     | RocketMQ Exporter |
+| Consume Qps                                | Msg/sec    | meter_rocketmq_broker_consume_qps                                       | The number of broker consumes messages per second.                                                     | RocketMQ Exporter |
```
Comment (wu-sheng) [inline docs/en/swip/SWIP-3.md:33]:
```suggestion
| Consume QPS                                | Msg/sec    | meter_rocketmq_broker_consume_qps                                       | The number of broker consumes messages per second.                                                     | RocketMQ Exporter |
```

File: docs/en/swip/SWIP-3.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+| Producer Offset                            | Count      | meter_rocketmq_topic_producer_offset                                    | The progress of a topic's production message.                                                          | RocketMQ Exporter |
+| Consumer Group Offset                      | Count      | meter_rocketmq_topic_consumer_group_offset                              | The progress of a topic's consumption message per consumer group.                                      | RocketMQ Exporter |
+| Producer Message Size                      | Byte       | meter_rocketmq_topic_producer_message_size                              | The size of messages produced per second.                                                              | RocketMQ Exporter |
+| Consumer Message Size                      | Byte       | meter_rocketmq_topic_consumer_message_size                              | The size of messages consumed per second.                                                              | RocketMQ Exporter |
```
Comment (wu-sheng) [inline docs/en/swip/SWIP-3.md:49]: Where are the Max offset metrics?

Comment (peachisai) [reply docs/en/swip/SWIP-3.md:49]: > Where are the Max offset metrics?

I originally thought that this metrics explanation did not need to be changed,it just change avg downsampling the max one ,ok, i will edit it

Comment (wu-sheng) [reply docs/en/swip/SWIP-3.md:49]: The list here are logic meaning, so, Max indicates slowest or something in the description.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**